packages/gestalt/src/InternalTextFieldIconButton.js

```diff
@@ -4,31 +4,36 @@ import classnames from 'classnames';
 import Box from './Box.js';
 import Pog from './Pog.js';
 import TapArea from './TapArea.js';
+import Tooltip from './Tooltip.js';
 import styles from './InternalTextField.css';
 import { TAB, SPACE, ENTER } from './keyCodes.js';
 
 type Props = {|
-  accessibilityClearButtonLabel?: string,
   accessibilityHidden?: boolean,
-  hoverStyle: 'default' | 'none',
+  accessibilityLabel?: string,
```

Comment:
Some usability changes here:
- no need to be overly prescriptive about how the a11y label is used
- make hoverStyle and pogPadding optional with defaults
- added accessibilityPressed and role ("switch") per recommendation for this type of on/off switch button

Reply:
I think here accessibilityPressed needs to be accessibilityChecked for role="switch" https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role